rllib/policy/sample_batch.py

```diff
@@ -832,6 +832,8 @@ def _slice(self, slice_: slice):
         """
         start = slice_.start or 0
         stop = slice_.stop or len(self)
```

Comment:
seems like this line plus the if block you just added is basically:

stop = math.min(slice_.stop, len(self))

and then maybe we can add a quick comment about capping it at len(self)?

Reply:
done